srv/salt/ceph/stage/deploy/core/default.sls

```diff
@@ -127,16 +127,17 @@ dashboard:
     - sls: ceph.dashboard
     - failhard: True
 
-mgr orchestrator module:
-  salt.state:
-    - tgt: {{ master }}
-    - tgt_type: compound
-    - sls: ceph.mgr.orchestrator
-    - pillar:
-        'salt_api_url': http://{{ master }}:8000/
-        'salt_api_username': admin
-        'salt_api_password': {{ salt.saltutil.runner('sharedsecret.show') }}
-    - failhard: True
+## disabled for now
```

Comment:
disable for now, cause the ds orchestrator doesn't provide the iscsi url which takes priority in the dashboard